routes/index.js

```diff
@@ -1,7 +1,23 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-
+const Movie = require("../models/Movie.model.js");
 /* GET home page */
-router.get('/', (req, res, next) => res.render('index'));
+router.get("/", (req, res) => res.render("index"));
+
+router.get("/movies", (req, res) => {
+  Movie.find({}).then((allMovies) => {
+    //console.log("Your movies are: ", allMovies);
+    res.render("movies", { allMovies });
+  });
+});
+
+router.get("/movies/:_id", (req, res) => {
+  const desiredMovie = req.params._id;
+  console.log("Your movie's id is: ", desiredMovie);
+  Movie.find({ _id: desiredMovie }).then((movieId) => {
```

Comment:
and in here you can name the parameter inside the .then( ) **oneMovie** or even the **desiredMovie** like you did above. Since this one is the variable that will hold the full movie object. Again, in the long run it helps a lot!